@here/harp-examples/src/datasource_features_lines-and-points.ts

```diff
@@ -247,7 +250,7 @@ export namespace LinesPointsFeaturesExample {
                     text-align: center;
                     font-family: monospace;
                     left: 50%;
```

Comment:
Alternatively, we can use the following CSS, as the current approach with absolute positioning and then adjustment with percentage is IMHO a bit strange:

```
  margin: 0 auto;
  font-size: 15px;
  padding-top: 10px;
```